index.html

```diff
@@ -30,7 +30,7 @@
  h2 {
 	font-size: 18px;
 	font-weight: normal;
-	margin-top: 4px;
+	margin-top: 0px;
```

Comment:
Remove. 0px is the default

Reply:
(I think, please confirm)

Reply:
Seems like 0px it is not the default, so I'm keeping this line for now.